Doc/library/asyncio-dev.rst

```diff
@@ -281,6 +285,7 @@ coroutine in another coroutine and use classic try/except::
     loop.close()
 
 Another option is to use the :meth:`loop.run_until_complete`
+(or alternatively, the provisional :meth:`asyncio.run`)
```

Comment:
```
:func:`asyncio.run`, not :meth:
```

Nit picking: I wouldn't focus too much on the fact that `asyncio.run` is provisional -- the only reason for it to be provisional is because we're unsure about some obscure edge cases. The function won't be removed and it's **the preferred way** of running an asyncio program (as it's almost impossible to use `loop.run_until_complete` API correctly :( ).

Reply:
Nit picking makes clear docs ;-)  Will update now.